coffee_core/src/nurse/strategy.rs

```diff
@@ -74,3 +74,39 @@ impl Handler for GitRepositoryLocallyAbsentStrategy {
         }
     }
 }
+
+/// Stategy for migration of the repository global directory
+/// to a local directory for each network, see [1]
+///
+/// This is a strategy tht return the list of network that
+/// needs to be migrated to to the new repository configuration.
+///
+/// [1] https://github.com/coffee-tools/coffee/issues/234
+pub struct CoffeeRepositoryDirCleanUp;
```

Comment:
Can we rename it to `CoffeeRepositoryDirCleanUpStrategy` to keep the pattern?

Reply:
Agree thanks